app.py

```diff
@@ -1493,17 +1493,19 @@ def modify_query(**new_values):
     return '{}?{}'.format(request.path, url_encode(args))
 
 
+# Todo TB: Re-write this somewhere sometimes following the line below
 # We only store this @app.route here to enable the use of achievements -> might want to re-write this in the future
 @app.route('/auth/public_profile', methods=['POST'])
 @requires_login
 def update_public_profile(user):
     body = request.json
+    print(body)
```

Comment:
is the print intentional?

Reply:
This should be removed, thanking for catching this issue!